scripts/mkcloudhost/boot.mkcloud

```diff
@@ -12,6 +12,6 @@ for n in $(seq 1 $cloudspernode) ; do
 done
 for x in D I ; do
     iptables -$x FORWARD -d 192.168.0.0/16 -j ACCEPT
-    iptables -t nat -$x POSTROUTING -s 10.164.0.0/15 -j MASQUERADE
+    iptables -t nat -$x POSTROUTING '!' -s 10.164.0.0/15 -j MASQUERADE
```

Comment:
Which IPs do you mean with "private" and "external" ?
IMHO, the outgoing 10.164 IPs need to be masqueraded to the host's addr, because they are not routed outside Nuremberg.
The 192.168* IPs should already be NATed by libvirt-created rules, so are not covered here.